bespiral.community/bespiral.community.hpp

```diff
@@ -138,14 +139,12 @@ class [[eosio::contract("bespiral.community")]] bespiral : public eosio::contrac
                       (track_stock)(quantity)(units));
   };
 
-  TABLE table_index {
-    std::uint64_t id;
-    std::uint64_t last_used_sale_id;
-    std::uint64_t last_used_objective_id;
-    std::uint64_t last_used_action_id;
-    std::uint64_t last_used_claim_id;
-
-    std::uint64_t primary_key() const { return id; }
+  TABLE indexes {
+    // Values based on the blockchain data as of 17th September 2019
+    std::uint64_t last_used_sale_id = 15;
+    std::uint64_t last_used_objective_id = 8;
+    std::uint64_t last_used_action_id = 33;
+    std::uint64_t last_used_claim_id = 0;
```

Comment:
What is the status for this?

Reply:
@lucca65 an action in the contract?

Reply:
yeah, something that allow us to set the values for this index table

Reply:
because this is hard coded to the ids used in the contract at the time you created this pull request, not only those values changed, but also we now have several different chains

Reply:
@lucca65 aaah I see what you mean that makes alot of sense, on it ...